modern/README.md

```diff
@@ -0,0 +1,193 @@
+
+### New Docker Strategy (June 2021)
+
+After many updates, new compiler releases, instability and incompatibility problems, we decided to clean the house, move a step forward with Conan Docker Tools. So far, we usually follow the same recipe, when a compiler version is released, we also release a new docker image, but we have two main problems:
+
+- Ubuntu doesn’t package new compiler versions for each distro release, so we need to use a new Ubuntu version as base image. As a consequence, non LTS versions become a problem when EOL comes.
+- Each Ubuntu release uses different package versions, including very important projects, like glibc, which results in incompatibility from a package created by distro version to another version.
+
+Given this priority (same base images, glibc and stdlibc++ versions for all) we will support only the compilers that we managed to build and run in the base image, the rest will be deprecated from ConanCenter because on this technical reasons.
+
+Second, we decided to use a single base image, not too old and not the latest, something in the middle, to keep a glibc still compatible for some old distro releases. Thus, we choose Ubuntu 16.04 LTS (Xenial) which its EOL is in 2024. When close to the EOL date, we can move to 18.04 and so on.
+
+As the compiler version is always a problem to align according to the distro release, we decided to build all compilers from sources, which includes GCC and Clang. Although this increased the build time considerably, we still now have full control over the compiler used in our images.
+
+We totally understand that companies and users are still using our “legacy” docker images, so we won’t deprecate them soon. This kind of change requires rebuilding all packages again, and depending on the number, it could take weeks. Thus, don’t worry, we won’t remove them from Docker hub.
+
+**TL;DR** New Docker images will use Ubuntu 16.04 as base and build all compilers from sources. Thus, all binaries generated will link with the same glibc and stdlibc++ versions.
+
+### Legacy Docker Images
+
+Legacy docker images will be moved to "legacy" folder and eventually their dockerfiles will no longer be maintained in this repository (EOL to be decided).
+
+### Official Docker Images
+
+These are the images uploaded to Docker Hub and currently used by [Conan Center](https://conan.io/center):
+
+#### GCC
+
+| Version                                                                            | Arch    |  Status, Life cycle  |
+|------------------------------------------------------------------------------------|---------|----------------------|
+| [conanio/gcc5-ubuntu16.04: gcc 5](https://hub.docker.com/r/conanio/gcc5-ubuntu16.04/)          | x86_64  |  Supported           |
```

Comment:
Should the table be updated?

```suggestion
| [conanio/gcc5-ubuntu16.04: gcc 5](https://hub.docker.com/r/conanio/gcc5-ubuntu16.04/)          | x86_64  |  Maintenance Only         |
```

Reply:
Sorry my ignorance, but what's the difference? I understand "Supported" as "maintained".

Reply:
For me there is some difference: supported means that we will _add features_ (new compilers, new CMake versions,... things like that) while maintenance means that we will only fix bugs, but no changes will be made otherwise.

However, this table is listing the new images, so I think `Supported` is better than `Maintenance Only` here, but I'm not a native English speaker, so maybe the misunderstanding is on my side.

Reply:
You are spot on, I saw "Legacy Docker Images" above the table but missed the following header

 https://github.com/conan-io/conan-docker-tools/pull/267/files#diff-679cc3486654be2732971ec20fa5b84d5348d5dc72f6cca5fe6f102515ba9ac6R31

Reply:
So we update the Legacy images to `Maintenance Only`, right?

Reply:
Okay, I'll do it in a next PR, I don't want to touch current files in CDT, we already have 48 changes in this PR, and losing an entire afternoon reviewing a single PR is not cool.